src/main/java/net/smartcosmos/dao/objects/impl/ObjectPersistenceService.java

```diff
@@ -39,6 +40,23 @@ public ObjectResponse create(String accountUrn, ObjectCreate createObject) {
     }
 
     @Override
+    public Optional<ObjectResponse> update(String accountUrn, ObjectResponse updateObject) {
```

Comment:
I don't think so, @jbauerrfid . What gets in, what gets out, and what gets stored are different Object objects for different purposes:

- `ObjectCreate` is the model for create requests
- `ObjectUpdate` is the model for update requests
- `ObjectResponse` is the model for any Object response
- `ObjectEntity` is the model for persistence in the database

Although, they look similar, they're not identical -- just think of the `accountUrn` field.

In my opinion, it makes sense not to mix them in a modular architecture.